shims/spark311/src/main/scala/org/apache/spark/sql/rapids/shims/spark311/GpuInMemoryTableScanExec.scala

```diff
@@ -28,85 +28,85 @@ import org.apache.spark.sql.execution.{LeafExecNode, SparkPlan}
 import org.apache.spark.sql.execution.columnar.InMemoryRelation
 import org.apache.spark.sql.vectorized.ColumnarBatch
 
-case class GpuInMemoryTableScanExec(
-  attributes: Seq[Attribute],
+case class GpuInMemoryTableScanExec(attributes: Seq[Attribute],
   predicates: Seq[Expression],
   @transient relation: InMemoryRelation) extends LeafExecNode with GpuExec {
 
-    override val nodeName: String = {
-      relation.cacheBuilder.tableName match {
-        case Some(_) =>
-          "Scan " + relation.cacheBuilder.cachedName
-        case _ =>
-          super.nodeName
-      }
+  override val nodeName: String = {
```

Comment:
A lot of this is white space changes, which makes it really hard to review. Could you revert the changes to just indentation please?

Reply:
This file only has indentation changes. I can revert and create a separate patch for it

Reply:
I would prefer a separate patch.